src/postgresql/pgolap.tcl

```diff
@@ -738,6 +760,7 @@ set degree_of_parallel \"$pg_degree_of_parallel\" ;# Degree of Parallelism
 set scale_factor $pg_scale_fact ;#Scale factor of the tpc-h schema
 set host \"$pg_host\" ;# Address of the server hosting PostgreSQL
 set port \"$pg_port\" ;# Port of the PostgreSQL Server
+set azure \"$pg_azure\";#Azure Type Connection
```

Comment:
if we end up adding this option, should it be in the UI as well?

Reply:
Yes, any options for the CLI should be included in the GUI options as well. This requires updating pgopt.tcl to add an additional row to the interface. 
Also any options added should be included in the connectpool options as well.

Reply:
Will look into it